### ISSUE TITLE
texi2dvi related install problem

### ISSUE THREAD

**Issue by armish** [frankkramer-lab/rBiopaxParser#2]
Hi,

I have recently came across your paper on Bioinformatics and was trying to install the package on my Mac OS X 1.7.X with R 2.15.1 installed. I followed the steps you have outlined in your _README.md_ file successfully, but the last install-from-GtiHub step fails with the following error:

```
* preparing 'rBiopaxParser':
* checking DESCRIPTION meta-information ... OK
* installing the package to re-build vignettes
* creating vignettes ... ERROR
Loading required package: graph
...

...
Adding to graph:  control - INHIBITION Controllers:  protein_A Controlleds:  protein_C
Adding to graph:  control - ACTIVATION Controllers:  protein_C Controlleds:  protein_D
Adding to graph:  control - INHIBITION Controllers:  protein_C Controlleds:  protein_E
Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
  Running 'texi2dvi' on 'rBiopaxParserVignette.tex' failed.
Calls: <Anonymous> -> texi2pdf -> texi2dvi
Execution halted
Error: Command failed (1)
```

Any ideas?

**Comment by frankkramer** [comment]
Hi,

thanks for trying out my package!
To be perfectly honest I haven't tried my package on MacOS yet, however 
your problem sounds like pdflatex or another tools to build the package 
vignette isnt found.
It seems like everything works on your machine except for (re-)building 
the pdf vignette from the tex document.
The easiest solution is that the package is now on Bioconductor 2.12 
(current devel version).
You can download the package at:
http://www.bioconductor.org/packages/devel/bioc/html/rBiopaxParser.html

Best wishes,
Frank

**Comment by armish** [comment]
Frank, thanks for reply!

As you have suggested, I tried the BioConductor way this time; and this is what I get when I follow the steps outlined on the web site:

```
$ R

R version 2.15.1 (2012-06-22) -- "Roasted Marshmallows"
Copyright (C) 2012 The R Foundation for Statistical Computing
ISBN 3-900051-07-0
Platform: x86_64-apple-darwin9.8.0/x86_64 (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type 'license()' or 'licence()' for distribution details.

  Natural language support but running in an English locale

R is a collaborative project with many contributors.
Type 'contributors()' for more information and
'citation()' on how to cite R or R packages in publications.

Type 'demo()' for some demos, 'help()' for on-line help, or
'help.start()' for an HTML browser interface to help.
Type 'q()' to quit R.

> source("http://bioconductor.org/biocLite.R")
Bioconductor version 2.11 (BiocInstaller 1.8.3), ?biocLite for help
> biocLite("rBiopaxParser")
BioC_mirror: http://bioconductor.org
Using Bioconductor version 2.11 (BiocInstaller 1.8.3), R version 2.15.
Installing package(s) 'rBiopaxParser'
Warning message:
package ‘rBiopaxParser’ is not available (for R version 2.15.1) 
```

any suggestions?

**Comment by frankkramer** [comment]
Hi,

sorry for confusing you here!
rBiopaxParser has only very recently been accepted to Bioconductor, it 
is not in the current release version (2.11) but will be included in the 
next release 2.12! However you can already install the package from the 
devel branch, either by downloading the package from

http://www.bioconductor.org/packages/devel/bioc/html/rBiopaxParser.html

or by using:

source("http://bioconductor.org/biocLite.R")
biocLite("rBiopaxParser", 
siteRepos="http://bioconductor.org/packages/2.12/bioc")

Best wishes,
Frank

Am 02.02.2013 00:08, schrieb B. Arman Aksoy:
